### PR TITLE
Yongbin4

### PR DESCRIPTION
**Pull request by limwenfeng**

### CODE REVIEW

File: src/pages/Groups/Group2/styles/Screen1.styled.ts [hunk 1]
```diff
@@ -17,25 +17,26 @@ export const Screen = styled.div`
...
   display: inline-block;
   padding-left: 5%;
   padding-right: 5%;
-  line-height: 10vh;
+  line-height: 8vh;
 `
 
 export const TextSection = styled.div`
-  height: 10vh;
+  height: 20vh;
```
Comment (limwenfeng) [inline src/pages/Groups/Group2/styles/Screen1.styled.ts]: This is not big enough. Amos wanted 3/5 of the screen. So it should be around 50vh to 60 vh

Comment (shaokiat) [reply src/pages/Groups/Group2/styles/Screen1.styled.ts]: I thought he said 2/5. I'll change it to 40vh

Comment (limwenfeng) [reply src/pages/Groups/Group2/styles/Screen1.styled.ts]: Oh okay

**Comment by limwenfeng** [comment]
LGTM!

**Pull request merged**